thankyou_letters/models/partner_communication.py

```diff
@@ -33,6 +33,20 @@ class PartnerCommunication(models.Model):
         domain=[('type', '=', 'sentence')])
     success_sentence = fields.Text(related='success_sentence_id.body_text')
     add_success_story = fields.Boolean(related='config_id.add_success_story')
+    amount = fields.Float(string='Amount', compute='_compute_donation_amount',
+                          store=True)
+
+    @api.multi
+    @api.depends('object_ids')
+    def _compute_donation_amount(self):
+        for communication in self:
+            model = communication.config_id.model
+            if model == 'account.invoice.line':
+                object_ids = map(int, communication.object_ids.split(','))
```

Comment:
Or better yet, you can use method `get_objects()` provided in partner_communication_job:
```suggestion
                invoice_lines = communication.get_objects()
```